src/FixedDataTableWidthHelper.js

```diff
@@ -22,6 +22,33 @@ function getTotalWidth(/*array*/ columns) /*number*/ {
   return totalWidth;
 }
 
+function getMaxVisibleColumns(/*array*/ columns, /*number*/ width) /*number*/ {
+  var maxColumns = 0;
+  var currentWidth = 0;
+
+  var scrollableColumns = columns.filter(function(column) {
+    var fixed = column.props.fixed;
+    if (fixed) {
+      width -= column.props.width;
```

Comment:
Added a comment. I didn't like a seperate loop pattern here because I have to re-fetch the fixed property